proto/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Protobuf API
+This folder contains all protobuf files that we have defined.
+We follow the Google Cloud's [API design guide](https://cloud.google.com/apis/design/), including the following conventions:
+
+* Ordering:
+  1. _syntax, package, import, option_ statements
+  2. overview documentation
+  3. _service_ definition(s)
+  4. request and response messages (ordered as respective methods)
+  5. resource messages while parents are defined before child resources
+* File names: 
+  * <lower_case_underscore_separated_names>.proto
+  * File name corresponding to (single) service
+* proto file only containing resources, consider naming this file simply as resources.proto
+* ENUM (Scale): The first value should be named ENUM_TYPE_UNSPECIFIED
+* Commenting: Comment services, RPCs and messages
+
```

Comment:
It´s missing if we always use response/request messages or only if >1 return values are present.

Reply:
I´ve updated the Readme.